### ISSUE TITLE
update examples to include schemaLocation

### ISSUE THREAD

**Issue by crotwell** [FDSN/StationXML#137]
Current examples look like:
```
<?xml version='1.0' encoding='UTF-8'?>
<FDSNStationXML xmlns="http://www.fdsn.org/xml/station/1" schemaVersion="1.1">
```
would be better to include the schemaLocation attr like:
```
<?xml version="1.0" encoding="ISO-8859-1"?>
<FDSNStationXML xmlns="http://www.fdsn.org/xml/station/1" 
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"  
    xsi:schemaLocation="http://www.fdsn.org/xml/station/1 http://www.fdsn.org/xml/station/fdsn-station-1.1.xsd" 
    schemaVersion="1.1">
```

Also update section 1.1.2 so show better first line.

**Comment by crotwell** [comment]
Oops, that should be UTF-8 like this:

```
<?xml version="1.0" encoding="UTF-8"?>
<FDSNStationXML xmlns="http://www.fdsn.org/xml/station/1" 
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"  
    xsi:schemaLocation="http://www.fdsn.org/xml/station/1 http://www.fdsn.org/xml/station/fdsn-station-1.2.xsd" 
    schemaVersion="1.2">
```

**Comment by crotwell** [comment]
PR #144

**Comment by crotwell** [comment]
merged #144
